app/src/main/res/layout/sheet_schedule_appointment.xml

```diff
@@ -156,5 +156,20 @@
       android:textColor="@color/white"
       app:buttonState="in-progress"
       tools:ignore="ButtonStyle" />
+
+    <com.google.android.material.button.MaterialButton
+      android:id="@+id/nextButton"
+      style="?attr/materialButtonStyle"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="@dimen/spacing_8"
+      android:layout_weight="1"
+      android:backgroundTint="?attr/colorSecondary"
```

Comment:
We don't have to set `backgroundTint` here, it will be derived from the `materialButtonStyle`